text/MateuszTokarz/opis.tex

```diff
@@ -0,0 +1,99 @@
+\documentclass[12pt]{article}
+
+\usepackage[utf8]{inputenc}
+\usepackage[T1]{fontenc}
+\usepackage{graphicx}
+\usepackage{xcolor} 
+\usepackage{amsthm}
+\usepackage{amsmath}
+
+\theoremstyle{plain}
+\newtheorem*{tw}{Twierdzenie}
+\newtheorem*{df}{Definicja}
+\newtheorem*{lemat}{Lemat}
+
+\begin{document}
+\title{Aproksymacja najkrótszego nadsłowa}
+\author{Mateusz Tokarz}
+\maketitle
+
+\section*{Opis Problemu}
+Problem najkrótszego nadsłowa w wersji decyzyjnej jest problemem NP zupełnym. W związku z tym pojawiało się wiele podejść do aproksymacji optymalizacyjnej wersji problemu. Tarhio i Ukkonen oraz Turner napisali dwie niezależne prace naukowe, w których opisano algorytmy aproksymacyjne dla rozważanego problemu. Mimo, że autorzy twierdzili, że skonstruowane przez nich algorytmy powinny być $2$-aproksymacyjne, to nie udało im się udowodnić żadnego ograniczenia. 
```

Comment:
Oh, that's nice that you invoked Tarhio i Ukkonen and Turner works - can you provide the article names for them?